contracts/util.sol

```diff
@@ -29,6 +50,16 @@ library RandomUtil {
         return num.sub(tenPercent).add(randomSeededMinMax(0, tenPercent.mul(2), seed));
     }
 
+    function plusMinus10PercentSeededFast(uint256 num, uint256 seed) internal pure returns (uint256) {
```

Comment:
Wouldn't it be useful to have percentage as parameter? For potential future use.

Reply:
It would be useful, but I would prefer to cross this bridge another time.
The 10% figure is the most common use case for us right now and we do save a tiny bit of overhead from having to support a percentage parameter.
What i would want to do, if in the future we use other percentage ranges (probably for Journey), is to have a separate function in this file for it.
Contract sizes are only grown by referenced library functions, meaning even if we have tons of functions in here that's not used by most of our files, their size will not grow by extending util.sol unless we actually reference them in the individual contracts.